### ISSUE TITLE
[a11y Design Review] Benefits - Dependent Experience Team: Review of 686c/674 Form Updates

### ISSUE THREAD

**Issue by sara-amanda** [department-of-veterans-affairs/va.gov-team#76841]
> [!CAUTION]
> - **Goal:** Provide _before_ Midpoint Review Friday 2/23/2024

## CAIA Accessibility Review of 686c/674 Form Updates

CAIA will review the prototype for accessibility issues and make recommendations to the Benefits - Dependent Experience Team. 

- **Kickoff Date:** 2/21/2024
- **Collab Cycle**: Yes (Midpoint 2/23/2024)

### Prototype
- **Review**: [Figma Designs V2](https://www.figma.com/file/7W55oNwdVXvXOTI9SaFzQ7/686c-Add-or-Remove-Dependents?type=design&node-id=319-47346&mode=design&t=iWU0j0ylBz1z2GED-0)
- **Recommend:** [Template Started for Design Review / a11y Findings](https://github.com/department-of-veterans-affairs/va.gov-team/blob/master/teams/CAIA/accessibility/Deliverables/Benefits%20Dependent%20Experience/202402-design-review-686c.md)

## Definition of Done

```[tasklist]
### A11y Tasks to Complete
- [x] [Created artifact draft outline 2/22/2024](https://github.com/department-of-veterans-affairs/va.gov-team/blob/master/teams/CAIA/accessibility/Deliverables/Benefits%20Dependent%20Experience/202402-design-review-686c.md)
- [x] Link to this ticket, and WIP artifact, in originating ticket
- [x] Review authenticated and unauthenticated views [Figma Designs V2](https://www.figma.com/file/7W55oNwdVXvXOTI9SaFzQ7/686c-Add-or-Remove-Dependents?type=design&node-id=319-47346&mode=design&t=iWU0j0ylBz1z2GED-0)
- [x] Complete [artifact](https://github.com/department-of-veterans-affairs/va.gov-team/blob/master/teams/CAIA/accessibility/Deliverables/Benefits%20Dependent%20Experience/202402-design-review-686c.md)
- [x] Update team about completed review via originating ticket and Slack prior to Midpoint 2/23
```

```[tasklist]
### Deliverable
- [x] Completed accessibility review 🔗
```

**Comment by EvanAtCoforma** [comment]
# Accessibility Feedback

## Presentation of content

1. From past AT user sessions, multiple users raised concerns about reading a “wall of text” compared to interacting with accordion styled content. Explore ways to condense the information for users who may be cognitively overwhelmed from the large amount of text.
2. **Consideration:**  Use the “one thing per page” concept. Each step can be contained with its own screen, allowing users to process each step individually. This should help certain users be able to focus on one step at a time, decreasing the risk of mental fatique.
3. **Consideration:**  Convert the list of instructions to an interactive checklist, as this formatting could help with identifying call to actions. A user may glance through the directions in a condensed list format, while a checklist with more spacing between  each item can create more direct interaction. It can also help users keep track of the information they gathered as they move through the directions.

## More than one CTA

1. For the "authenticated in progress" screen, the "Continue your application" CTA should be repeated at the end of the directions so users wouldn't have to scroll back to the top to continue their application.
2. The same concept applies for the "start a new application" CTA, as a user would need to scroll back up to activate that function.
3. This will also provide consistency throughout related screens, as the unauthenticated and authenticated screens have the repeated CTA at the end of the instructions.

## Clear callout

1. For the “in progress saved” screen, users could benefit from a text alert that says “progress has been saved” above the blue information box, or positioned somewhere obvious. While the current screen provides information for when the progress was last saved, it seems less present than saying “your progress has been saved.” After this text, the current text in the screen can be read to the user.

**Comment by ajialeilani** [comment]
I have updated the [mockup](https://www.figma.com/file/7W55oNwdVXvXOTI9SaFzQ7/686c-Add-or-Remove-Dependents?type=design&node-id=319%3A46756&mode=design&t=M6mvuzdvYBiOvgT9-1) with the above suggestions

**Comment by ajialeilani** [comment]
Reconfirming I've looked at these and implemented.
